include/PriorityQueue.h

```diff
@@ -5,15 +5,44 @@
 #include <exception>
 
 
-// any templates?
+// comparator that takes two template variables and returns the difference between them
+template <class T>
+struct MyComparator {
+	T operator()(T a, T b) const {
+		return a - b;
+
+	}
+};
+
```

Comment:
good code but i can't see the dog files, make sure to include them in the pr

Reply:
I do not understand why that has happened, I added files for you to evaluate.

Reply:
I apologize for that, i don't understand why it  happened.
I added missing files.
 Please let me know if you encounter any further difficulties viewing the files.